### PR TITLE
Makes Weed Killer actually Hurt Venus Human Traps

### PR DESCRIPTION
**Pull request by JamieD1**
What is in this PR:
Man eating plants now take damage from plant b gone grenades

:cl:  
rscadd: Man eating plants take damage from plant b gone.
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 1]
```diff
@@ -255,6 +255,8 @@
 			if(!C.wear_mask) // If not wearing a mask
 				var/damage = min(round(0.4*reac_volume, 0.1),10)
 				C.adjustToxLoss(damage)
+		if(istype(M, /mob/living/simple_animal/hostile/venus_human_trap)
```
Comment (adamsong) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]:
```suggestion
		if(istype(M, /mob/living/simple_animal/hostile/venus_human_trap))
```
Do better

**Review by adamsong — DISMISSED** [review]

**Comment by Marmio64** [comment]
Too much dmg imo

**Comment by SomeguyManperson** [comment]
do they even have reagents to process chemicals from

**Review by alexkar598 — DISMISSED** [review]
template

**Review by Bibby0110 — DISMISSED** [review]
Template + please confirm that you intend any spray volume to cause 10 damage.

**Comment by JamieD1** [comment]
Need to make it actually work.

**Comment by JamieD1** [comment]
If someone could make this would, I would be appreciated.

**Comment by JamieD1** [comment]
SOMEONE FIX MY PR!

REEEEEEEEEEEEEEEEEEEEE

**Comment by JamieD1** [comment]
I fixed it, please review.

**Review by alexkar598 — COMMENTED** [review]
template

**Pull request merged**